2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_setup.py:_flush():81] Current SDK version is 0.21.3
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_setup.py:_flush():81] Configure stats pid to 250057
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_setup.py:_flush():81] Loading settings from /home/akos.prucs@egroup.hu/.config/wandb/settings
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_setup.py:_flush():81] Loading settings from /home/akos.prucs@egroup.hu/LLMEnergyEfficiency/wandb/settings
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_init.py:setup_run_log_directory():686] Logging user logs to /home/akos.prucs@egroup.hu/LLMEnergyEfficiency/wandb/run-20250911_140042-gwm56p2d/logs/debug.log
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_init.py:setup_run_log_directory():687] Logging internal logs to /home/akos.prucs@egroup.hu/LLMEnergyEfficiency/wandb/run-20250911_140042-gwm56p2d/logs/debug-internal.log
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_init.py:init():813] calling init triggers
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_init.py:init():818] wandb.init called with sweep_config: {}
config: {'model': 'Qwen/Qwen2.5-0.5B', 'dataset': 'mmlu', 'style': 'cot', 'think_budget': 0, 'K': 0, 'dtype': 'bfloat16', '_wandb': {}}
2025-09-11 14:00:42,619 INFO    MainThread:250057 [wandb_init.py:init():854] starting backend
2025-09-11 14:00:42,825 INFO    MainThread:250057 [wandb_init.py:init():857] sending inform_init request
2025-09-11 14:00:42,828 INFO    MainThread:250057 [wandb_init.py:init():865] backend started and connected
2025-09-11 14:00:42,831 INFO    MainThread:250057 [wandb_init.py:init():936] updated telemetry
2025-09-11 14:00:42,838 INFO    MainThread:250057 [wandb_init.py:init():960] communicating run to backend with 90.0 second timeout
2025-09-11 14:00:43,486 INFO    MainThread:250057 [wandb_init.py:init():1011] starting run threads in backend
2025-09-11 14:00:43,582 INFO    MainThread:250057 [wandb_run.py:_console_start():2494] atexit reg
2025-09-11 14:00:43,582 INFO    MainThread:250057 [wandb_run.py:_redirect():2342] redirect: wrap_raw
2025-09-11 14:00:43,583 INFO    MainThread:250057 [wandb_run.py:_redirect():2411] Wrapping output streams.
2025-09-11 14:00:43,583 INFO    MainThread:250057 [wandb_run.py:_redirect():2434] Redirects installed.
2025-09-11 14:00:43,585 INFO    MainThread:250057 [wandb_init.py:init():1057] run started, returning control to user process
2025-09-11 14:01:36,424 INFO    wandb-AsyncioManager-main:250057 [service_client.py:_forward_responses():84] Reached EOF.
2025-09-11 14:01:36,424 INFO    wandb-AsyncioManager-main:250057 [mailbox.py:close():137] Closing mailbox, abandoning 1 handles.
2025-09-11 14:01:36,479 ERROR   wandb-AsyncioManager-main:250057 [asyncio_manager.py:fn_wrap_exceptions():183] Uncaught exception in run_soon callback.
Traceback (most recent call last):
  File "/home/akos.prucs@egroup.hu/LLMEnergyEfficiency/.venv/lib/python3.12/site-packages/wandb/sdk/lib/asyncio_manager.py", line 181, in fn_wrap_exceptions
    await fn()
  File "/home/akos.prucs@egroup.hu/LLMEnergyEfficiency/.venv/lib/python3.12/site-packages/wandb/sdk/lib/service/service_client.py", line 65, in _send_server_request
    await self._writer.drain()
  File "/home/akos.prucs@egroup.hu/.local/share/uv/python/cpython-3.12.11-linux-x86_64-gnu/lib/python3.12/asyncio/streams.py", line 392, in drain
    await self._protocol._drain_helper()
  File "/home/akos.prucs@egroup.hu/.local/share/uv/python/cpython-3.12.11-linux-x86_64-gnu/lib/python3.12/asyncio/streams.py", line 166, in _drain_helper
    raise ConnectionResetError('Connection lost')
ConnectionResetError: Connection lost
